_drafts/SC4-1-2-role-has-required-states-and-properties.md

```diff
@@ -0,0 +1,77 @@
+---
+name: Role has required states and properties
+description: | Elements that has a semantic role must also specify all required states and properties
+  
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+Any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has a [semantic role](#semantic-role). 
+
+### Expectation 1
+
+For each test target all [required states and properties](https://www.w3.org/TR/wai-aria/#requiredState) for that role as specified by [WAI-ARIA](https://www.w3.org/TR/wai-aria) are present.
+
+### Expectation 2
+
+Each of the [required states and properties](https://www.w3.org/TR/wai-aria/#requiredState) have a valid value according to the [value type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value) for the given state or property as specified by [WAI-ARIA](https://www.w3.org/TR/wai-aria).
```

Comment:
[required states and properties]
- I see no definition for this. It should be a part of this PR?